src/engine/Eval_generic.ml

```diff
@@ -318,6 +359,8 @@ and eval_str _env ~code v =
     | Int i -> string_of_int i
     | Float f -> string_of_float f
     | String s -> s
+    | Date (y, m, d) ->
+        string_of_int y ^ "-" ^ string_of_int m ^ "-" ^ string_of_int d
```

Comment:
Could you use a format string instead?

Try `Common.spf "%s-%s-%s" (string_of_int y) (string_of_int m) (string_of_int d)`

Reply:
Done!